test/test.py

```diff
@@ -0,0 +1,132 @@
+import pickle
+from biocypher import BioCypher
+import pytest
+import yaml
+import importlib
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+def convert_input_labels(label, replace_char="_"):
+    return label.replace(" ", replace_char)
+
+def parse_schema(bcy):
+    schema = bcy._get_ontology_mapping()._extend_schema()
+    edges_schema = {}
+    node_labels = set()
+
+    for k, v in schema.items():
+        if v["represented_as"] == "edge": 
+            edge_type = convert_input_labels(k)
+            #Check if there are source and target fields
+            source_type = v.get("source", None)
+            target_type = v.get("target", None)
+            if source_type is not None and target_type is not None:
+                if isinstance(v["input_label"], list):
+                    label = convert_input_labels(v["input_label"][0])
+                    source_type = convert_input_labels(source_type[0])
+                    target_type = convert_input_labels(target_type[0])
+                else:
+                    label = convert_input_labels(v["input_label"])
+                    source_type = convert_input_labels(source_type)
+                    target_type = convert_input_labels(target_type)
+
+                output_label = v.get("output_label", None)
+                edges_schema[label.lower()] = {"source": source_type.lower(), "target":
+                    target_type.lower(), "output_label": output_label.lower() if output_label is not None else None}
+
+        elif v["represented_as"] == "node":
+            label = v["input_label"]
+            if isinstance(label, list):
+                label = label[0]
+            label = convert_input_labels(label)
+            node_labels.add(label)
+
+    return node_labels, edges_schema
+    
+
+@pytest.fixture(scope="session")
+def setup_class(request):
+    # Load schema config
+    bcy = BioCypher(schema_config_path='config/schema_config.yaml', biocypher_config_path='config/biocypher_config.yaml')
+    node_labels, edges_schema = parse_schema(bcy) 
+   
+    # Load adapters config
+    adapters_config_path = request.config.getoption("--adapters-config")
+    dbsnp_rsids = request.config.getoption("--dbsnp-rsids")
+    dbsnp_pos = request.config.getoption("--dbsnp-pos")
+    logging.info("Loading dbsnp rsids map")
+    dbsnp_rsids_dict = pickle.load(open(dbsnp_rsids, 'rb'))
+    logging.info("Loading dbsnp pos map")
+    dbsnp_pos_dict = pickle.load(open(dbsnp_pos, 'rb'))
+   
+    # Load adapters config
+    with open(adapters_config_path, 'r') as f:
+        adapters_config = yaml.safe_load(f)
+
+    return node_labels, edges_schema, adapters_config, dbsnp_rsids_dict, dbsnp_pos_dict
+
+@pytest.mark.filterwarnings("ignore")
+class TestBiocypherKG:
```

Comment:
Can you add few lines of comment explaining what the tests do (or exactly what's being tested) and what output they expect?